docs/diagrams/UiClassDiagram.puml

```diff
@@ -11,8 +11,8 @@ Class UiManager
 Class MainWindow
 Class HelpWindow
 Class ResultDisplay
-Class PersonListPanel
-Class PersonCard
+Class XYZListPanel
+Class XYZCard
 Class StatusBarFooter
 Class CommandBox
```

Comment:
Could you add one more class `KeyboardShortcuts` that `MainWindow` has a dependency on?